examples/src/main/java/io/github/tcdl/msb/examples/FacetsAggregator.java

```diff
@@ -86,7 +86,7 @@ public static void main(String[] args) throws ScriptException, FileNotFoundExcep
                     responder.send(responsePayload);
                 });
 
-                requester.publish(request);
+                requester.publish(request, responder.getOriginalMessage().getCorrelationId());
```

Comment:
I think that author of microservices should know about correlationId,as for me previous approach when we keep correlationId inside ResponderImpl is better.